test/core/regressions.js

```diff
@@ -99,7 +99,7 @@ describe("Core htmx Regression Tests", function(){
         byId("d1").innerText.should.equal('Replaced');
     });
 
-    it('does not submit with a false condition on a form', function() {
+    it('submission is aborted but event default is not prevented when event is excluded via filter', function() {
```

Comment:
Is this a chance in behavior that could cause possible regressions?

Reply:
I would say no. If by any chance you rely on the event excluded by the filter preventing the default of that event you rely on a "bug" behavior.

There could be a case where someone is using a filter on a <a> element and relying on the behavior that any clicks even those excluded by filter does not navigate the user away. In this case one could say it's a regression, but I'd argue:

Why are you using an <a> element with href attr if you don't intend to navigate the user? Perhaps replace it with a button.

It'd be an extremely rare regression worth the change to allow ctrl/meta + click links by only adding a filter.

Reply:
what about someone who has an event filter on a form that has an hx-post on it?

if this change is accepted, the form would submit if the filter was false, no?

i see what you are saying here, but I think that this change might be a little too aggressive.